Dockerfile

```diff
@@ -1,23 +1,16 @@
 FROM python:3.6-alpine
 
-RUN apk --no-cache add \
-    python3 \
-    git \
-    libpq \
-    wget \
-    libffi \
-    libffi-dev \
-    ca-certificates \
-    python3-dev \
-    postgresql-dev \
-    build-base \
-    bash \
-    curl
```

Comment:
We need bash and curl to make the [`entrypoint.sh`](https://github.com/openspending/os-conductor/blob/198bf5c0223818422a053453566545afd56c30cb/docker/docker-entrypoint.sh) script work.

Reply:
okay

Reply:
bash and curl - check!